README.md

```diff
@@ -46,7 +46,28 @@ A COGS plugin is a folder in the `plugins` directory of a COGS Project. The plug
 
 ## Getting started
 
-If creating new COGS Media Master custom content or a new COGS Plugin we recommend using Vite and Typescript to create your project.
+If creating new COGS Media Master custom content or a new COGS Plugin we recommend using our template which includes [Turborepo](https://turborepo.com/), [Vite](https://vite.dev/), [TypeScript](https://www.typescriptlang.org/) and [ESLint](https://eslint.org/).
+
+### Using the Template
+
+Create a new project using the example template:
+
+```bash
+npx create-turbo@latest --example https://github.com/clockwork-dog/cogs-sdk/tree/main/template
+```
+
+You can then look at the README of the generated project to generate custom content or a plugin.
+
+```sh
+npx turbo generate
+yarn turbo generate
+pnpm turbo generate
+```
```

Comment:
I find it's handy to put these in separate code blocks because GitHub gives you a handy copy-to-clipboard button. There's not much point copying them all in one go!